src/asynqp/__init__.py

```diff
@@ -19,8 +19,12 @@
 def connect(host='localhost',
             port=5672,
             username='guest', password='guest',
-            virtual_host='/', *,
-            loop=None, sock=None, **kwargs):
+            virtual_host='/',
+            *,
+            loop=None,
+            sock=None,
+            nodelay=True,
```

Comment:
If you want to implement good flexible api, provide function that allow to operate on socket (i.e. on protocol and transport).

Passing list of args for `setsockopt` is not good solution. Next time, user wants to switch `O_CLOEXEC` (which is not set using `setsockopt()`) and so on. Flexible API should allow to access socket directly (as asyncio does).

I don't know how to do that in your library. So I provide quick-and-not-so-dirty method to speedup 10X without any API change and in a way that cannot hurt at any condition. That's why I don't actually want default parameter, but add it because you forced that.